### PR TITLE
SOAR-8106 Update workflow tags

### PR DESCRIPTION
**Pull request by tcheng-r7**
Almost all workflows have had their use case tags updated in their workflow.spec.yaml. Additionally, separate hub tags for vendors and third-party products have been added.

All title and description attributes are also consistent with the use of double quotes.

Formatting Rundown for workflow.spec.yaml relevant to PR:

products: list of double quoted strings
name: unquoted string
title: double quoted string
description: double quoted string
hub_tags:
  use_cases: list of unquoted strings or list with empty string
  keywords: list of unquoted strings or list with empty string
  features: list of unquoted strings or list with empty string
  vendors: list of unquoted strings or list with empty string
  products: list of unquoted strings or list with empty string

## Requirements

Workflow contributors, verify you have completed the following items by checking them off:

### Checklist
- [ ] Workflow verified as working in a test
- [ ] Workflow bundle includes `Name_of_Workflow.icon` file, `help.md` file, `workflow.spec.yaml` file, and `screenshots` directory
- [ ] Workflow title should follow general [Title of Articles](http://grammar.yourdictionary.com/capitalization/rules-for-capitalization-in-titles.html) format
- [ ] Workflow includes a detailed Description
- [ ] Workflow step names follow [Title of Articles](http://grammar.yourdictionary.com/capitalization/rules-for-capitalization-in-titles.html) format
- [ ] Triggers, Decisions, and other workflow steps include descriptions where needed
- [ ] Workflow includes at least one `Final Report` artifact that summarizes workflow execution details
- [ ] Workflow bundle (directory) is named after the title of the workflow in snake case with first letters capitalized, e.g. `workflows/Name_of_Workflow/`
- [ ] `help.md` file includes Description, Key Features, Requirements, Setup, Technical Details, Version History, and References
- [ ] `workflow.spec.yaml` file includes `products`, `name`, `title`, `description`, `use_cases` tags, `keywords` tags, `source_url`, and the `name` for each screenshot matches a PNG image in the `screenshots` directory

### Assessment

Post your results with screenshots in this PR using the following style:

```
### Screenshots

Screenshots help your reviewers provide valuable feedback on your contribution. To add a screenshot here, simply drag and drop the PNG files from a file explorer.

#### Workflow Builder

<Insert Workflow Image Here>

#### Job

<Insert Job Image Here>

#### Artifact

<Insert Artifact Image Here>
```

### CODE REVIEW

File: workflows/Create_BMC_Incident_for_Top_Remediations_from_InsightVM/workflow.spec.yaml [hunk 1]
```diff
@@ -8,9 +8,11 @@ vendor: rapid7
 support: rapid7
 status: []
 hub_tags:
-  use_cases: [devops, remediation_management]
-  keywords: [bmc]
+  use_cases: [ticketing, vulnerability_management]
+  keywords: []
```
Comment (mrinehart-r7) [inline workflows/Create_BMC_Incident_for_Top_Remediations_from_InsightVM/workflow.spec.yaml:12]: @chogan-r7 do we want to add any keywords here?

File: workflows/Create_Incident_in_ServiceNow_from_Slack/workflow.spec.yaml [hunk 2]
```diff
@@ -2,15 +2,17 @@ extension: workflow
...
+description: "Automatically create an incident in ServiceNow directly from Slack."
 version: 1.0.0
 vendor: rapid7
 support: rapid7
 status: []
 hub_tags:
-  use_cases: [remediation_management, threat_detection_and_response]
-  keywords: [servicenow, ticketing]
+  use_cases: [ticketing, vulnerability_management]
+  keywords: []
```
Comment (mrinehart-r7) [inline workflows/Create_Incident_in_ServiceNow_from_Slack/workflow.spec.yaml:12]: @chogan-r7 do we want to add any keywords here?

File: workflows/Hello_IDR_Alert/workflow.spec.yaml [hunk 3]
```diff
@@ -2,15 +2,17 @@ extension: workflow
...
 vendor: rapid7
 support: rapid7
-status: ["hidden"]
+status: [hidden]
 hub_tags:
-  use_cases: [threat_detection_and_response]
+  use_cases: [utility, alerting_and_notifications]
   keywords: [cloud_enabled]
   features: []
+  vendors: []
```
Comment (mrinehart-r7) [inline workflows/Hello_IDR_Alert/workflow.spec.yaml:14]: @chogan-r7 This should include `rapid7` right?

File: workflows/Hello_IDR_Alert/workflow.spec.yaml [hunk 4]
```diff
@@ -2,15 +2,17 @@ extension: workflow
...
+  products: []
```
Comment (mrinehart-r7) [inline workflows/Hello_IDR_Alert/workflow.spec.yaml:15]: @chogan-r7 this should include `insightidr` right?

File: workflows/InsightVM_Top_25_Remediations_to_JIRA/workflow.spec.yaml [hunk 5]
```diff
@@ -1,16 +1,18 @@
...
+description: "Create and update Jira issues based on top risk-reducing remediation steps from InsightVM."
 version: 1.0.4
 vendor: rapid7
 support: rapid7
 status: []
 hub_tags:
-  use_cases: [vulnerability_management]
-  keywords: [vulnerability_management, insightvm, jira]
+  use_cases: [ticketing, vulnerability_management]
+  keywords: []
```
Comment (mrinehart-r7) [inline workflows/InsightVM_Top_25_Remediations_to_JIRA/workflow.spec.yaml:12]: @chogan-r7 do we want to add any keywords here?

File: workflows/InsightVM_Top_25_Remediations_to_Zendesk/workflow.spec.yaml [hunk 6]
```diff
@@ -8,9 +8,11 @@ vendor: rapid7
 support: rapid7
 status: []
 hub_tags:
-  use_cases: [vulnerability_management]
-  keywords: [insightvm, zendesk, vulnerability_management]
+  use_cases: [ticketing, vulnerability_management]
+  keywords: []
```
Comment (mrinehart-r7) [inline workflows/InsightVM_Top_25_Remediations_to_Zendesk/workflow.spec.yaml:12]: @chogan-r7 do we want to add any keywords here?

File: workflows/Quarantine_Endpoint_with_Broadcom_Symantec_Endpoint_Protection_from_Microsoft_Teams/workflow.spec.yaml [hunk 7]
```diff
@@ -1,16 +1,19 @@
 extension: workflow
 products: ["insightconnect"]
 name: Quarantine_Endpoint_with_Broadcom_Symantec_Endpoint_Protection_from_Microsoft_Teams
-title: "Quarantine Endpoint with Broadcom Symantec Endpoint Protection from Microsoft Teams"
-description: Manage the quarantine state of an endpoint with Broadcom Symantec Endpoint Protection from Microsoft Teams.
+title: "Quarantine Endpoint with Broadcom Symantec Endpoint Protection from Microsoft"
+  Teams
```
Comment (mrinehart-r7) [inline workflows/Quarantine_Endpoint_with_Broadcom_Symantec_Endpoint_Protection_from_Microsoft_Teams/workflow.spec.yaml:5]:
```suggestion
title: "Quarantine Endpoint with Broadcom Symantec Endpoint Protection from Microsoft Teams"
```

File: workflows/URLScan_with_Slack/workflow.spec.yaml [hunk 8]
```diff
@@ -2,15 +2,17 @@ extension: workflow
 products: ["insightconnect"]
 name: URLScan_with_Slack
 title: "Enrich URL with urlscan.io from Slack"
-description: This workflow enrichs a URL given via Slack command in urlscan.io and reports information back to Slack
+description: "This workflow enrichs a URL given via Slack command in urlscan.io and reports information back to Slack"
```
Comment (mrinehart-r7) [inline workflows/URLScan_with_Slack/workflow.spec.yaml:5]: This should read `This workflow enriches ...`, fixing the typo in `enrichs`

File: workflows/Virtually_Patch_Vulnerabilities_Found_by_InsightVM_with_Deep_Security/workflow.spec.yaml [hunk 9]
```diff
@@ -32,4 +34,4 @@ resources:
   - name: artifact01.png
     title: Report Artifact Sucessful Assignment of One Rule
   - name: artifact02.png
-    title: Report Artifact Sucessful Assignment of Multiple Rules
\ No newline at end of file
+    title: Report Artifact Sucessful Assignment of Multiple Rules
```
Comment (mrinehart-r7) [inline workflows/Virtually_Patch_Vulnerabilities_Found_by_InsightVM_with_Deep_Security/workflow.spec.yaml]: Can you fix the typo here in `Sucessful`? It should read `Successful`

**Review by mrinehart-r7 — CHANGES_REQUESTED** [review]
Tagging in @chogan-r7 for some content questions and @tcheng-r7 for some slight fixes to the changes here.

**Review by mrinehart-r7 — DISMISSED** [review]

File: workflows/Create_Incident_in_ServiceNow_from_Slack/workflow.spec.yaml (hunk 2 above)
Comment (chogan-r7) [reply workflows/Create_Incident_in_ServiceNow_from_Slack/workflow.spec.yaml:12]: Nope idc that much about the keyword category

File: workflows/Create_BMC_Incident_for_Top_Remediations_from_InsightVM/workflow.spec.yaml (hunk 1 above)
Comment (chogan-r7) [reply workflows/Create_BMC_Incident_for_Top_Remediations_from_InsightVM/workflow.spec.yaml:12]: Nope idc that much about the keyword category

File: workflows/Hello_IDR_Alert/workflow.spec.yaml (hunk 3 above)
Comment (chogan-r7) [reply workflows/Hello_IDR_Alert/workflow.spec.yaml:14]: yes good call

File: workflows/Hello_IDR_Alert/workflow.spec.yaml (hunk 4 above)
Comment (chogan-r7) [reply workflows/Hello_IDR_Alert/workflow.spec.yaml:15]: yes perfect

File: workflows/InsightVM_Top_25_Remediations_to_JIRA/workflow.spec.yaml (hunk 5 above)
Comment (chogan-r7) [reply workflows/InsightVM_Top_25_Remediations_to_JIRA/workflow.spec.yaml:12]: no need!

**Review by mrinehart-r7 — APPROVED** [review]

**Review by mberezin-r7 — APPROVED** [review]

**Pull request merged**